packages/node/README.md

```diff
@@ -34,15 +34,27 @@ npm install @imgly/background-removal-node
 
 ## Usage
 
-```typescript
-import {removeBackground} from "@imgly/background-removal-node";
-// const {removeBackground} = require("@imgly/background-removal-node");
-
-let image_src: ImageData | ArrayBuffer | Uint8Array | Blob | URL | string = ...;
+```js
+const { removeBackground } = require('@imgly/background-removal-node');
+const fs = require('fs');
```

Comment:
there is 'fs/promises'. I think that's better then fs.promises usage later

Reply:
Changed